### PR TITLE
RDoc-2823 Cloud - Support form -> remove severity, extend response section

### PR DESCRIPTION
**Pull request by PFYasu**
Ticket: https://issues.hibernatingrhinos.com/issue/RDoc-2823/Cloud-Support-form-remove-severity-extend-response-section

### CODE REVIEW

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown [hunk 1]
```diff
@@ -65,6 +66,10 @@ The *Cloud Support Activation* email requests the enabling of the **Cloud Suppor
 If you want to change this period, please send a *support request* via the *Support form*.
 {INFO/}
 
+{INFO: }
+Cloud Support activation takes up to **3 working days**.
```
Comment (PFYasu) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown:70]: 3 business days?

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 2]
```diff
@@ -45,34 +45,23 @@ For a full support options comparison please check the [Pricing](https://cloud.r
...
+| **Response**      | **Description**                                                                                                                                                                                                                                                              |
+|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Standard          | Non-critical errors, general questions, requests for enhancements to the Software.                                                                                                                                                                                           |
+| Next business day | Reply within a next business day. The error is not critical - no data has been lost, and the system has not failed. The error has been identified and does not prevent normal operation, or the situation may be temporarily circumvented using an available workaround.     |
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:59]:
```suggestion
| Next business day | Reply within a next business day. Use when the error is not critical - no data has been lost, and the system has not failed. The error has been identified and does not prevent normal operation, or the situation may be temporarily circumvented using an available workaround.     |
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 3]
```diff
@@ -45,34 +45,23 @@ For a full support options comparison please check the [Pricing](https://cloud.r
...
+| Within 2 hours    | Your query will be responded to within two hours. Due to an error in a production environment, the Software is down or seriously impacted, or there is no workaround currently available, or the workaround is cumbersome to use, or the Customer data is lost or destroyed. |
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:60]:
```suggestion
| Within 2 hours    | Your query will be responded to within two hours. Use when due to an error in a production environment, the Software is down or seriously impacted, or there is no workaround currently available, or the workaround is cumbersome to use, or the Customer data is lost or destroyed. |
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 4]
```diff
@@ -45,34 +45,23 @@ For a full support options comparison please check the [Pricing](https://cloud.r
...
-
-| **Response** | **Description**                                   |
-|--------------|---------------------------------------------------|
-| Standard     | Reply within a day.                               |
-| Immediate    | Your query will be responded to within two hours. |
-
-{INFO: }
-**Immediate Response** is only available for Customers with the **Cloud Support option**.  
-**Immediate Response** is not available for **Free tier** instances.
+**Within 2 hours** response time is only available for Customers with the **Cloud Support option**.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:63]:
```suggestion
**Within 2 hours** response time is only available for Customers with the **Cloud Support** support option.  
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown [hunk 5]
```diff
@@ -45,34 +45,23 @@ For a full support options comparison please check the [Pricing](https://cloud.r
...
-| **Response** | **Description**                                   |
-|--------------|---------------------------------------------------|
-| Standard     | Reply within a day.                               |
-| Immediate    | Your query will be responded to within two hours. |
-
-{INFO: }
-**Immediate Response** is only available for Customers with the **Cloud Support option**.  
-**Immediate Response** is not available for **Free tier** instances.
+**Within 2 hours** response time is only available for Customers with the **Cloud Support option**.  
+**Within 2 hours** response time is only available for **Production tier** instances.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-support.markdown:64]:
```suggestion
**Within 2 hours** response time is only available for **Production tier** related issues.
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown (hunk 1 above)
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown:70]:
```suggestion
Cloud Support activation takes up to **3 business days**.
```

**Review by gregolsky — COMMENTED** [review]
see my comments

**Review by gregolsky — APPROVED** [review]

**Pull request merged**